### PR TITLE
style(*): remove linting errors

### PR DESCRIPTION
**Pull request by tykowale**
Update linting to match editorconfig and fix any current issues

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "SeedReactProject",
-    "private": true,
+    "private": false,
```
Comment (weblancaster) [inline package.json]: Since it's not being published on npm we can leave it true.

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
         "css-loader": "^0.25.0",
         "eslint": "^3.0.0",
         "eslint-loader": "^1.0.0",
-        "eslint-plugin-react": "^6.0.0",
+        "eslint-plugin-react": "^6.7.0",
```
Comment (weblancaster) [inline package.json]: What's the reason for this outdate?

**Comment by weblancaster** [comment]
Thanks a lot for the PR @tykowale

File: package.json (hunk 2 above)
Comment (tykowale) [reply package.json]: I was trying to fix a `jsx-uses-vars` linting error and I forced it up to 6.7 to see if that helped. Plus I was mixed up between `~` and `^` so I'll revert this

**Comment by tykowale** [comment]
No problem, I needed a start pack for react and figured I would check out what you used. Thanks for building this

**Comment by weblancaster** [comment]
@tykowale awesome.. feel free to improve. thanks.

**Pull request merged**